Added FAQ

siri-chandana-macha opened the pull request:
Fixed #21

Implemented an FAQ section to enhance user experience by providing quick answers to common questions about interacting with AI chatbots.

**Note:** Make Sure to add all labels in #21
 
![image](https://github.com/user-attachments/assets/3317de76-917b-4ae5-b90a-c336dfc3be5c)

[Kv Nexus .webm](https://github.com/user-attachments/assets/44fd15d5-9723-4fef-bc23-ff274f1c20a5)

kvcops commented:
can you add a font style and a font color for the answers in the Q&A section

siri-chandana-macha commented:
Ok . which font type and color?

On Sat, 26 Oct, 2024, 5:19 pm KARRI VAMSI KRISHNA, ***@***.***>
wrote:

> can you add a font style and a font color for the answers in the Q&A
> section
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kvcops/KV-Nexus/pull/38#issuecomment-2439548721>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BK7ODR6HJYND5YNAQVR7K2DZ5N6UTAVCNFSM6AAAAABQUWWANKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZZGU2DQNZSGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

kvcops commented:
> Ok . which font type and color?
> […](#)
> On Sat, 26 Oct, 2024, 5:19 pm KARRI VAMSI KRISHNA, ***@***.***> wrote: can you add a font style and a font color for the answers in the Q&A section — Reply to this email directly, view it on GitHub <[#38 (comment)](https://github.com/kvcops/KV-Nexus/pull/38#issuecomment-2439548721)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BK7ODR6HJYND5YNAQVR7K2DZ5N6UTAVCNFSM6AAAAABQUWWANKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZZGU2DQNZSGE> . You are receiving this because you were mentioned.Message ID: ***@***.***>

something that suits for the ui and themes...make them animated(if possible)

siri-chandana-macha commented:
Have a look I have changed Font style and color and added a hover effect

![Screenshot 2024-10-26 184850](https://github.com/user-attachments/assets/fc36270d-27c5-4087-bda8-f6b910a3d173)

![image](https://github.com/user-attachments/assets/d5a48cd1-89a8-4485-9fff-aa85b55b3281)

kvcops commented:
the container backrgound and the theme doesnt seems to match ...also you can remove the last FAQ question "What benefits do i gain from using the Quantum Speed and AI synergy"
